specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/fetch/isolation.read.committed.aborted/server.rpt

```diff
@@ -73,6 +73,8 @@ write zilla:begin.ext ${kafka:beginEx()
                                .build()}
 write flush
 
+write await SENT_ABORT
+
```

Comment:
Should this be between `partition(0, 3, 3, 3)` and `partition(0, 4, 4, 4)` instead?

Reply:
Not sure if I understood you correctly, it is between those two message

Reply:
I don't mean the `beginEx`, referring to between the `flushEx` and the `dataEx`.

Reply:
test are failing if I put between the `flushEx` and the `dataEx`.

Reply:
Yes, makes sense - the barrier needs to go before the flush.